File: CurrencyAnalyzerFrontend/src/app/dtos/currency-record-difference-response-dto.ts
export class CurrencyRecordDifferenceResponseDto{

  date: string;

  bidPriceDifference: number;

  salePriceDifference: number;

  constructor() {
      this.date = '';
      this.bidPriceDifference = 0;
      this.salePriceDifference = 0;
  }

}

File: CurrencyAnalyzerBackend/src/main/java/com/example/currencyanalyzerbackend/currencyRecordDifference/dto/CurrencyRecordDifferenceResponseDto.java
package com.example.currencyanalyzerbackend.currencyRecordDifference.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class CurrencyRecordDifferenceResponseDto {

    private String date;

    private Double bidPriceDifference;

    private Double salePriceDifference;

}
